It's not a fix for CVE-2015-4495, but should make user feel himself a bit more easy
--- mozilla-esr24/browser/app/profile/firefox.js.1	2015-09-14 16:08:46.393326487 +0300
+++ mozilla-esr24/browser/app/profile/firefox.js	2015-09-14 16:09:07.547561634 +0300
@@ -1228,7 +1228,7 @@
 // Completely disable pdf.js as an option to preview pdfs within firefox.
 // Note: if this is not disabled it does not necessarily mean pdf.js is the pdf
 // handler just that it is an option.
-pref("pdfjs.disabled", false);
+pref("pdfjs.disabled", true);
 // Used by pdf.js to know the first time firefox is run with it installed so it
 // can become the default pdf viewer.
 pref("pdfjs.firstRun", true);
